packages/rlc-common/src/buildClient.ts

```diff
@@ -228,7 +240,8 @@ function isSecurityInfoDefined(
 
 export function getClientFactoryBody(
   model: RLCModel,
-  clientTypeName: string
+  clientTypeName: string,
+  isMultipleCredential: boolean
```

Comment:
I would suggest to instead of adding this as a positional parameter to add an options bag. This will help us prevent having a lot of positional parameters if we need to keep adding

```ts
interface GetClientFactoryOptions {
   isMultipleCredential: boolean;
}
export function getClientFactoryBody(
  model: RLCModel,
  clientTypeName: string,
  options?: GetClientFactoryBodyOptions
)

```

Reply:
updated.